sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/samples/Utils.java

```diff
@@ -2581,6 +2581,10 @@ public static void print(RoleDefinition role) {
             for (String notAction : permission.notActions()) {
                 builder.append("\n\t\t\tName :").append(notAction);
             }
+            builder.append("\n\t\tPermission Data Actions: " + permission.dataActions().size());
+            for (String notAction : permission.dataActions()) {
```

Comment:
`for (String notAction : permission.dataActions())` -> `for (String dataAction : permission.dataActions())`

Reply:
Changed, thanks!